lib/gui-utils/src/GUIIcons.h

```diff
@@ -62,4 +62,85 @@ static const unsigned char PROGMEM ic_sensor_live[] = {
     B10101010,
     B01010100,
     B00000000,
-};
\ No newline at end of file
+};
+
+#ifdef Emoticons
+  static unsigned char Smileface1[] = {                                     //32x32
```

Comment:
Please change to expression name. The numbers 1, 2, 3 does not mean nothing 
```suggestion
  static unsigned char SmilefaceHappy[] = {                                     //32x32
```
for example.

Reply:
Resolved

Reply:
Expression name changed:
https://github.com/danielbernalb/esp32-hpma115s0/commit/e15cc4bab6ad5edeec8961e4e06e03d533c9c7d5